integrations/unstructured/src/haystack_integrations/components/converters/unstructured/converter.py

```diff
@@ -39,22 +55,21 @@ def __init__(
         progress_bar: bool = True,  # noqa: FBT001, FBT002
     ):
         """
-        :param api_url: URL of the Unstructured API. Defaults to the hosted version.
-            If you run the API locally, specify the URL of your local API (e.g. http://localhost:8000/general/v0/general).
-            See https://unstructured-io.github.io/unstructured/api.html#using-the-api-locally for more information.
-        :param api_key: API key for the Unstructured API (https://unstructured.io/#get-api-key).
+        :param api_url: URL of the Unstructured API. Defaults to the URL of the hosted version.
+            If you run the API locally, specify the URL of your local API (e.g. `"http://localhost:8000/general/v0/general"`).
+        :param api_key: API key for the Unstructured API.
+            It can be explicitly passed or read the environment variable `UNSTRUCTURED_API_KEY` (recommended).
             If you run the API locally, it is not needed.
-            If you use the hosted version, it defaults to the environment variable UNSTRUCTURED_API_KEY.
         :param document_creation_mode: How to create Haystack Documents from the elements returned by Unstructured.
-            - "one-doc-per-file": One Haystack Document per file. All elements are concatenated into one text field.
-            - "one-doc-per-page": One Haystack Document per page.
-               All elements on a page are concatenated into one text field.
-            - "one-doc-per-element": One Haystack Document per element.
-              Each element is converted to a Haystack Document.
+        `"one-doc-per-file"`: One Haystack Document per file. All elements are concatenated into one text field.
+        `"one-doc-per-page"`: One Haystack Document per page.
+        All elements on a page are concatenated into one text field.
+        `"one-doc-per-element"`: One Haystack Document per element. Each element is converted to a Haystack Document.
```

Comment:
Bulleted lists inside a specific parameter do not play well with `pydoc-markdown`

Reply:
Hmm, this should've been an enum instead of a string literal. We should open an issue to convert this (later).